Sources/StreamVideoSwiftUI/Utils/AudioValuePercentageNormaliser.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The normaliser computes the percentage values or value of the provided array or value.
+internal class AudioValuePercentageNormaliser {
+    internal let valueRange: ClosedRange<Float> = -50...0
```

Comment:
where does -50 comes from?

Reply:
`AVAudioRecorder` [average power meter](https://developer.apple.com/documentation/avfaudio/avaudiorecorder/1387176-averagepower) returns values from -160(absolute) to 0(peak volume for microphone). However, values below `-50` are still very quite for us (this is called the noiseFloor). We are using `-50` here as this is the value we are using for async messages and is based on an average value between online references to implementing audio recording as also other recording SDKs implementations ([DSWaveForm](https://github.com/dmrschmidt/DSWaveformImage/blob/f46cf0626b5abcb6ed2eb47b554dd5e31ee18d35/Sources/DSWaveformImage/WaveformAnalyzer.swift#L20) one of the best waveform rendering SDKs is using -50 as its noise-floor).